src/pipeline_migration/migrate.py

```diff
@@ -124,10 +126,12 @@ def resolve_pipeline(pipeline_file: FilePath) -> Generator[FilePath, Any, None]:
             os.close(fd)
             pipeline = {"spec": spec["pipelineSpec"]}
             dump_yaml(temp_pipeline_file, pipeline)
+            origin_checksum = file_checksum(temp_pipeline_file)
```

Comment:
A few lines above you are dumping pipeline spec without style. Will it lose the indentation?

Reply:
It's an intermediate file used for applying migrations.

Reply:
I see now, original indentation doesn't matter since you are comparing hash before and after migration.